src/lang/modifyAst/addFillet.test.ts

```diff
@@ -439,6 +441,115 @@ const extrude001 = extrude(50, sketch001)
   })
 })
 
+const runModifyAstWithFilletAndTagTest = async (
+  code: string,
+  selectionSnippets: Array<string>,
+  radiusValue: number,
+  expectedCode: string
+) => {
+  // ast
+  const astOrError = parse(code)
+  if (err(astOrError)) {
+    return new Error('AST not found')
+  }
+  const ast = astOrError as Program
```

Comment:
is this `as` needed? I know this is just a test, but best to avoid in general.
Plus the return type for parse should be `Program | Error` so your type narrowing should work just fine?